packages/element-app/src/components/Pages/DIDProfilePage/DIDProfilePage.js

```diff
@@ -2,32 +2,36 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Typography from '@material-ui/core/Typography';
 import Grid from '@material-ui/core/Grid';
-import LinearProgress from '@material-ui/core/LinearProgress';
 import { Pages } from '../../index';
 
 import { DIDDocument } from '../../DIDDocument';
 
+import { Loading } from '../../Loading/Loading';
+
 import { CreateDefaultDID } from '../../CreateDefaultDID';
 import { DIDDocumentEditorBar } from '../../DIDDocumentEditorBar';
 import { SidetreeOperation } from '../../SidetreeOperation';
 
 export class DIDProfilePage extends Component {
   componentWillMount() {
-    if (!this.props.wallet.data || !this.props.wallet.data.keys) {
+    if (
+      !this.props.keystore.keystore.data ||
```

Comment:
The page crashes before this line on my machine. Need to add  `!this.props.keystore.keystore ||` as the first condition

Reply:
excellent catch